### PR TITLE
feat: Apply finschia-sdk 0.49.1

### PR DESCRIPTION
**Pull request by tkxkd0159**
## Description

closes: #XXXX

- [x] Bump Go from 1.20 to 1.22
- [x] Update outdated go pkgs
- [x] Update outdated github actions
- [x] Remove rosetta
- [x] Apply the same lint rules as `finschia-sdk`

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/finschia/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/finschia/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/format.yml [hunk 1]
```diff
@@ -1,35 +0,0 @@
-name: Check Make Format
```
Comment (zemyblue) [inline .github/workflows/format.yml:1]: Is it no problem to remove this checking?
This ci action is added in #230.

Comment (zemyblue) [reply .github/workflows/format.yml:1]: As I know, the purpose of #230 is to check the imports order. 
Does the new linter and lint rules also check this imports order?
And the lint rules should avoid checking `statik.go` imports order.

**Review by jaeseung-bae — DISMISSED** [review]

**Comment by zemyblue** [comment]
Please check goreleaser.yaml and `make release`.

**Comment by zemyblue** [comment]
> Please check goreleaser.yaml and `make release`.

Please apply following diff to fix gorelease error.
gorelease error is occurred by gorelease version is updated to v2.

```
diff --git a/.goreleaser.yml b/.goreleaser.yml
index 8d770a3..4b26a0f 100644
--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -1,5 +1,7 @@
 project_name: fnsad
 
+version: 2
+
 env:
   - CGO_ENABLED=1
 
@@ -108,5 +110,3 @@ checksum:
 snapshot:
   name_template: SNAPSHOT-{{ .Commit }}
 
-changelog:
-  skip: false
diff --git a/Makefile b/Makefile
index da348ce..a05cc2a 100644
--- a/Makefile
+++ b/Makefile
@@ -310,7 +310,7 @@ release-dry-run:
 		release \
 		--clean \
 		--release-notes ./RELEASE_NOTE.md \
-		--skip-publish
+		--skip=publish
 
 release-snapshot:
 	docker run \
@@ -326,7 +326,7 @@ release-snapshot:
 		--clean \
 		--release-notes ./RELEASE_NOTE.md \
 		--snapshot \
-		--skip-validate \
-		--skip-publish
+		--skip=validate \
+		--skip=publish
 
 .PHONY: release release-dry-run release-snapshot
```

**Review by jaeseung-bae — APPROVED** [review]

**Review by 0Tech — APPROVED** [review]
LGTM

**Pull request merged**